Formula/eiffelstudio.rb

```diff
@@ -17,20 +17,32 @@ class Eiffelstudio < Formula
   depends_on "pkg-config" => :build
   depends_on "gtk+"
 
+  uses_from_macos "pax" => :build
+
   def install
-    system "./compile_exes", "macosx-x86-64"
-    system "./make_images", "macosx-x86-64"
+    # Fix flat namespace usage in C shared library.
+    if OS.mac?
+      system "tar", "xf", "c.tar.bz2"
+      inreplace "C/CONFIGS/macosx-x86-64",
+        "-flat_namespace -undefined suppress",
+        "-undefined dynamic_lookup"
+      system "tar", "cjf", "c.tar.bz2", "C"
+    end
+
+    os = OS.mac? ? "macosx" : OS.kernel_name.downcase
+    system "./compile_exes", "#{os}-x86-64"
```

Comment:
`x86-64` everywhere seems weird.

Reply:
Should we maybe just use something like `os_tag = "#{os}-x86-64"`?  And then use that variable everywhere instead?

Reply:
We could, but I guess my point was that it's `x86-64` even on ARM.

Reply:
`brew audit` will fail if we've been doing that.

Reply:
Hmm, so should we be changing anything?.  It's certainly odd and confusing but I'm not sure anything can be done right now.